gdal/gcore/gdaldataset.cpp

```diff
@@ -4578,6 +4578,30 @@ OGRLayerH GDALDatasetExecuteSQL( GDALDatasetH hDS,
             pszDialect));
 }
 
+
+/************************************************************************/
+/*                        GDALDatasetAbortSQL()                       */
+/************************************************************************/
+
+/**
+ \brief Abort any SQL statement running in the data store.
```

Comment:
```suggestion
 \brief Abort any SQL statement running in the data store.
 
 This function can be safely called from any thread (pending that the dataset object is still alive). Driver implementations will make sure that it can be called in a thread-safe way.
 
 This might not be implemented by all drivers. At time of writing, only SQLite, GPKG and PG drivers implement it
```

to add to the doc of the C++ method as well